nexus/src/app/sagas/region_replacement_drive.rs

```diff
@@ -929,21 +928,25 @@ async fn srrd_drive_region_replacement_prepare(
                         );
 
                         match &state {
-                            VmmState::Running | VmmState::Rebooting => {
-                                // Propolis server is ok to receive the volume
-                                // replacement request.
+                            VmmState::Running
+                            | VmmState::Rebooting
+                            | VmmState::Starting => {
+                                // Propolis server is expected to be there
+                                // (eventually, in the case of "Starting"), and
```

Comment:
I believe that a VMM isn't in the `Starting` state unless the Propolis server process does exist, FWIW --- if memory serves, that's the distinction between `Starting` and `Creating`. It may not yet be incarnating an instance, though.